### PR TITLE
Bookbinder 2, Boogaloo.

### PR DESCRIPTION
**Pull request by Mechoid**
Fix the Bookbinder not handling multiple page books.

### CODE REVIEW

**Comment by TheFurryFeline** [comment]
🤔 @ Travis's screeching about 

> code/modules/mob/living/silicon/robot/robot.dm:959:error: =: conflicts with previous definition
code/modules/mob/living/inventory.dm:148:error: =: conflicts with previous definition
polaris.dmb - 2 errors, 0 warnings (11/7/19 11:47 pm)

File: code/modules/library/lib_machines.dm [hunk 1]
```diff
@@ -526,17 +526,34 @@ datum/borrowbook // Datum used to keep track of who has borrowed what when and f
...
+	if(istype(O, /obj/item/weapon/paper) || istype(O, /obj/item/weapon/paper_bundle))
+		if(istype(O, /obj/item/weapon/paper))
+			user.drop_item()
+			O.loc = src
+			user.visible_message("[user] loads some paper into [src].", "You load some paper into [src].")
+			src.visible_message("[src] begins to hum as it warms up its printing drums.")
+			sleep(rand(200,400))
+			src.visible_message("[src] whirs as it prints and binds a new book.")
+			var/obj/item/weapon/book/b = new(src.loc)
+			b.dat = O:info
```
Comment (Atermonera) [inline code/modules/library/lib_machines.dm]: while the istype means that you can use O in this manner, this syntax is why communicator watches break so many glove-related things. It really should be explicitly casted to the target type

**Comment by Atermonera** [comment]
Tested?

**Comment by Mechoid** [comment]
Yes, though I'm perpetually concerned about Travis.

**Comment by Atermonera** [comment]
Sorry for taking so long to actually get back to this, school was stressful and busy.

**Pull request merged**